src/Sarif/Core/FileData.cs

```diff
@@ -48,19 +43,6 @@ public static FileData Create(Uri uri, bool computeHashes, out string fileDataKe
                             },
                         };
             }
-            //else if (files.Count == 1)
-            //{
-            //    fileData.Uri = uri;
-            //    fileDataKey = fileDataKey + "#" + fileData.Uri.ToString();
-            //}
-            //else
-            //{
-            //    Debug.Assert(!uri.IsAbsoluteUri);                    
-            //    fileData.Uri = uri;
-            //    fileDataKey = fileDataKey + fileData.Uri.ToString();
-            //}
-
-            //files.Add(fileData);
```

Comment:
This code can't be resurrected; there's no "files" variable in this method any more.